integration.go

```diff
@@ -53,11 +54,12 @@ type IntegrationConnection struct {
 }
 
 type AWSIntegrationInput struct {
```

Comment:
It seems this struct is duplicative of whats in `input.go` ?  why?

Reply:
Yeah I noticed that... not sure what happened here but didn't want to yank out the duplicate before figuring out which is the "good" one

Reply:
Lets make sure we follow this up